apix-impl/src/main/java/eu/xenit/apix/alfresco/configuration/ConfigurationServiceImpl.java

```diff
@@ -60,7 +61,7 @@ public Configurations getConfigurationFiles(String searchDirectory, String nameF
         }
         String[] searchDirectoryParts = searchDirectory.split("/");
 
-        NodeRef rootFolder = fileFolderService.getDataDictionary();
+        NodeRef rootFolder = AuthenticationUtil.runAsSystem(() -> fileFolderService.getDataDictionary());
```

Comment:
I would like a comment like _Running as system so user does not need read access on the folders inbetween_

Reply:
et voilà